src/model/ActivityCreationResponse.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2025-present Coinbase Global, Inc.
+ *
+ * This file is generated by Openapi Generator https://github.com/openapitools/openapi-generator
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *  Do not edit the class manually.
+ */
+
+import { ActivityType } from './enums/ActivityType';
+
+export type ActivityCreationResponse = {
+  activityType: ActivityType;
```

Comment:
I think we need to have two different ActivityType enums. This is the ActivitiesService ActivityType

You can see that kind in the response here: https://docs.cdp.coinbase.com/prime/reference/primerestapi_getportfolioactivity

The other kind is the one from Custody found in this AddressBook response: https://docs.cdp.coinbase.com/prime/reference/primerestapi_createportfolioaddressbookentry

Reply:
this is auto generated. and this is namespaced on how they are renamed into the request and response objects via types file.

Reply:
Right but it's importing the same activitytype here: https://github.com/coinbase-samples/prime-sdk-ts/blob/main/src/model/CreatePortfolioAddressBookEntryResponse.ts

Those are two different enums per the docs.

Reply:
yes updated to separate them. really don't like the custody/prime prepend, but is what it is